astropy/table/table.py

```diff
@@ -740,10 +740,9 @@ def show_in_browser(self,
         jsviewer : bool
             If `True`, prepends some javascript headers so that the table is
             rendered as a https://datatables.net data table.  This allows
-            in-browser searching & sorting.  See `JSViewer
-            <http://www.jsviewer.com/>`_
+            in-browser searching & sorting.
         jskwargs : dict
-            Passed to the `JSViewer`_ init.
+            Passed to the `astropy.table.jsviewer.JSViewer`_ init.
```

Comment:
Ah yes, I added JSViewer to the API docs. I can put this in a separate PR if you prefer.